src/NavBar/NavBar.jsx

```diff
@@ -1,12 +1,27 @@
-import React from 'react'
+import React, { useMemo, cloneElement } from 'react'
 import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import css from '@styled-system/css'
 
 import { Flex } from '@ivoryio/kogaio'
 import { NavItem } from './Item'
 
-export const NavBar = ({ children }) => <Container>{children}</Container>
+export const NavBar = ({ children }) => {
+  const content = useMemo(() => {
+    if (!children.length) {
+      return null
```

Comment:
shouldn't we return the Container here just with empty content ?

Reply:
That's exactly what we do.